content/en/wiki/time-value-of-money.md

```diff
@@ -0,0 +1,59 @@
+# Time Value of Money
```

Comment:
@reo-sam the [YAML front matter block](https://content.nuxtjs.org/writing/#front-matter) is missing :)

Here's how it looks for the DD page -
```YAML
---
title: Due Diligence (DD) checklist
category: wiki
position: 12
description: A checklist to reference while performing due diligence on a company.
---
```

Reply:
@reo-sam share a description not exceeding 70 words, I'll add it

Reply:
done.